src/components/shared/Tabs.jsx

```diff
@@ -0,0 +1,59 @@
+import classnames from 'classnames'
+import React, { useState } from 'react'
+
+import classNames from './Tabs.scss'
+
+export const Tabs = ({ children, initialTab = 0, ...props }) => {
+  const [tab, setTab] = useState(children[initialTab])
+
+  return (
+    <section className={classNames.root}>
+      <TabSelector
+        setTab={setTab}
+        tab={tab}
+        tabs={children}
+      />
+      <TabContent tab={tab} />
+    </section>
+  )
+}
+
+const TabSelector = ({ setTab, tab, tabs, ...props }) => {
+  const onClick = tab => () => {
+    setTab(tab)
+  }
+
+  return (
+    <section className={classNames.selector}>
+      <div className={classNames.selectorRail} />
+      {tabs.map(t => (
+        <TabLabel
+          active={t.props.label === tab.props.label}
+          label={t.props.label}
+          onClick={onClick}
```

Comment:
I think if we did 

```jsx
onClick={() => setTab(t)}
```

we could simplify both components (see comment in `TabLabel`).

Also, are we passing the entire ReactJS component here to `setTab`? If so, can we do this with tab ids instead? I'm not sure whether ReactJS guarantees component references will stay the same, if they change between renders (even if it's the same component) an equality check could fail. And we'd be leaking access to component internals.